### PR TITLE
Jenkins page updated

### PR DESCRIPTION
**Pull request by anisha147**

### CODE REVIEW

**Comment by lakhanmandloi** [comment]
Hello @anisha147,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://425.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/knowledge-platform-object-schema/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @anisha147,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://425.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/keycloak_admin_config_settings/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/sso_otpsms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/installation/mobile_app_forms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/mobile-app installation/install_mobile_setup/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/keycloak_user_federation/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/knowledge-platform-object-schema/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @anisha147,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://425.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/keycloak_admin_config_settings/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/sso_otpsms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/installation/mobile_app_forms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/mobile-app installation/install_mobile_setup/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/keycloak_user_federation/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/knowledge-platform-object-schema/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @anisha147,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://425.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/keycloak_admin_config_settings/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/sso_otpsms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/installation/mobile_app_forms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/mobile-app installation/install_mobile_setup/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/keycloak_user_federation/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/knowledge-platform-object-schema/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @anisha147,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://425.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/keycloak_admin_config_settings/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/sso_otpsms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/installation/mobile_app_forms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/mobile-app installation/install_mobile_setup/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/keycloak_user_federation/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/knowledge-platform-object-schema/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @anisha147,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://425.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/keycloak_admin_config_settings/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/sso_otpsms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/installation/mobile_app_forms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/mobile-app installation/install_mobile_setup/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/keycloak_user_federation/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/knowledge-platform-object-schema/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br />  <br/> **Modified sections are -** <br />

File: developer-docs/server-installation/keycloak_user_federation.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+allowSearch: true
+--- 
+
+## Deployment Steps for Keycloak User Federation
+
+To create a Keycloak user federation, ensure to take a backup of the Keycloak database and then complete the following steps:
+
+1.	Log in to the admin console using admin account and click the User Federation tab in the navigation pane as shown here. The User Federation page opens. 
+
+    <img src='images/keycloak_user_federation.png'/>
```
Comment (Sunitashelar) [inline developer-docs/server-installation/keycloak_user_federation.md]: Move images in proper folder. globally for all files in keycloak

**Comment by lakhanmandloi** [comment]
Hello @anisha147,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://425.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/keycloak_admin_config_settings/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/sso_otpsms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/installation/mobile_app_forms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/mobile-app installation/install_mobile_setup/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/keycloak_user_federation/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/knowledge-platform-object-schema/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @anisha147,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://425.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/keycloak_admin_config_settings/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/sso_otpsms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/installation/mobile_app_forms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/mobile-app installation/install_mobile_setup/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/keycloak_user_federation/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/knowledge-platform-object-schema/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br />  <br/> **Modified sections are -** <br />

**Comment by lakhanmandloi** [comment]
Hello @anisha147,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://425.qa.docs.sunbird.org/2.0.0/ .<br/> **Note:** Only _2.0.0_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/keycloak_admin_config_settings/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/configuring_sunbird/sso_otpsms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/installation/mobile_app_forms/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/mobile-app installation/install_mobile_setup/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/keycloak_user_federation/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/knowledge-platform-object-schema/ <br /> - http://425.qa.docs.sunbird.org/2.0.0/developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables/ <br />  <br/> **Modified sections are -** <br />

**Pull request merged**